public/app/plugins/panel/graph/specs/align_y.jest.ts

```diff
@@ -0,0 +1,200 @@
+import { alignYLevel } from '../align_yaxes';
```

Comment:
Change the name of this file to align_yaxes.jest.ts to align with the implementation file.